cornice_swagger/converters/__init__.py

```diff
@@ -7,11 +7,8 @@
 from cornice_swagger.converters.parameters import ParameterConversionDispatcher
 
 
-def convert_schema(schema_node):
-
-    dispatcher = TypeConversionDispatcher()
+def convert_schema(schema_node, dispatcher=TypeConversionDispatcher()):
```

Comment:
Now we have references to the dispatchers in the `CorniceSwagger` class, I would just remove these two and call the dispatchers directly from there.